### DISCUSSION TITLE
Aircraft Cards - Premade statistics and special traits

### DISCUSSION THREAD

**Discussion by asmialek** [Wollivan/SquadronLeader#67] [Ideas]
I was thinking about adding (as a module of optional rules) a collection of aircraft cards that would reference real WW1 planes. The plan is to be able to replace the "plane creation" process with this. Like in usual figure games, they would have a set of statistics and a small trait.

An example:
![fokker](https://user-images.githubusercontent.com/14300968/171136151-fb450b17-1524-466a-8aba-415380089a3b.png)

This ofc raises a few questions:
1. How to attach the statistics to real-world planes? Should we move from the standard 2,3,3,4 array?
2. How hard it will be to balance the planes? (Is this an engineering pun?) I guess for a game with multiple planes per player one could assign point values or some other recruiting schema, but I also wanted the cards to be viable for single-plane games.
3. How to handle other plane classes (partially as suggested in https://github.com/Wollivan/SquadronLeader/issues/11)? Afaik the division could be scout/fighter/bomber (although scout/fighter might be too blurry).

Let's make this a brainstorm 😄 What are your thoughts? I am already excited about the possibility of making the cards in different settings, such as WW2 or [REDACTED] (or even designing some custom sci-fi or dieselpunk settings)!

**Comment by asmialek** [comment]
I already have some more ideas for the traits:
- **Synchronization Gear**: This plane can reroll 1 additional die when attacking.
-  **Rotary Engine**: Maneuvers to the left cost 1 more Speed, but maneuvers to the right cost 1 less.
-  **Lightweight Fuselage**: When climbing gain 1 additional altitude token.

Btw, "Aircraft Blueprints" is a far better name for the thing than "Aircraft Cards" 😄

**Comment by lifelike** [comment]
This looks promising!

One thing that I think as an old print'n'player is that cards that are standard magic/poker size (2.5x3.5 inches in funny US measurements) are way more convenient (and cheap) to print out.  Feels more like a real game when you have them printed and in sleeves with some backing. So my vote would be to try to format the cards for that standard size.

You might want to have a look at the Inkscape script I created that I used for the 2D countersheet. It can be set up to make sheets of cards (in any size...). Or I could help out with that maybe. Just have to draw a template and put some text and image filenames in a spreadsheet.

Other thing is that it might be good to not mention specific Dis**y intellectual properties. I think what was meant above was to consider different settings "such as WW2 or Generic Space Opera"?

**Reply by asmialek** [reply]
Sure thing, I was just doing some quick mock-up, but the end product would be a standard tgc card size. I'll check out your script, it's a good opportunity to finally learn how Inkscape handles scripting.

Aaaand I've changed the THING in my original comment. True, better to be on the safe side 😂

**Comment by lifelike** [comment]
2. Balance. I think point costs are needed for multi-plane balance, and presumably if two players just want a two-plane dogfight they can pick planes with the same point cost? It would be weird to try to come up with super-powerful abilities for some really bad early-war plane to make it balanced against some great late-war plane, instead of just accepting that the former is bad and just give it a lower point cost?

**Reply by asmialek** [reply]
I agree that some points would be necessary. Maybe it could be as simple as having, for example, three tiers - one/two/three start or sth? And then in multi-plane battles, one could agree on how many points they can spend. This way it would keep the points concise enough to allow to say something like "let's have one play game with tier one aircrafts".

**Comment by lifelike** [comment]
One fun thing I saw this weekend was laser-cut frames for stat-cards. It was for some naval wargame I forgot the name of right now. But you put the stat card for a ship in a standard frame, and the frame had notches for tracking values like remaining healt (hull points or whatever). Makes me think that maybe those cards for plane stats could be combined with the status trackers that someone made, if there was room to place the status card in like a frame.

(This might contradict my comment above about using standard card sizes, if this is easier to do with a bigger blueprint. :) )

**Reply by lifelike** [reply]
I searched for what the frames was like, and it was something like this:
https://empiresatwar.co.uk/product/28mm-cruel-seas-double-card-holders/

It wasn't actually that exact model, but the idea is the same. Maybe that is something common in minis games, but I just saw it for the first time a few days ago and thought it was neat. Would be something 100% optional of course, but might influence the way things are printed on the cards.

**Reply by asmialek** [reply]
Yeah, I love the pin to track health in that status trackers made by @amnos0405. I think it could all fit on a standard game card, I'll fiddle with the template when I have some time.

**Comment by Wollivan** [comment]
Oh man this is exciting and I have MANY thoughts, so here goes!

First of all @asmialek this is brilliant and I love it! This is something X-Wing TMG has so I do like the unique planes idea and it feeling more real. It also creates a lot of scope for the campaign mode that is being worked on, where players can earn abilities to use in future missions.

On @lifelike 's point about the size of cards. I can't agree more! I have 800 MTG sleeves just _waiting_ to have plane stat cards in ;)

On the subject of point costs: Absolutely each plane will need a cost and I agree about having the balance not worry about WWI planes vs WWII planes. The WWI planes will just cost less because they suck comparatively.
A lesson we should learn from is the X-Wing 2.0 reboot they did, where the cards became very unbalanced, and they had to errata the rules/costs of many cards. So when they rebooted to 2.0 version of the game, the points costs of cards were no longer printed, but available online (and through the companion app) so if a certain ship became too good in tournaments, they could just nerf it without making everyones physical copy out of date.
We should have a thing about how we want to do this.

I can't express how joyful the idea of Sci-Fi/steam-punk ships and the not to be named Mouse Company owned ships makes me!


The biggest thing about this will be play testing for balance of abilities. But I do think this means you can move away from the standard 2,3,3,4 for ships that have stats. Because they can be balanced with cost.

Last thing:
This should be an alternate way of playing. I still want the entry level version of this game to be as simple as is possible, and then the progression onto Unique ships, campaigns and different types of ships even more exciting for players!

**Comment by asmialek** [comment]
The talks about balancing and campaign make me think - do we want to be able to have multiple traits on a plane? The way I initially envisioned it was that it was one plane - one trait. The argument for that would be, once again, simplicity.

But on the other hand, having multiple traits would help with the historical (and technical) accuracy. Afaik Fokker E. would have both a **Rotary Engine** and a **Synchronization Gear**. This would allow for more diversity and more customisation (like buying multiple upgrades for your aircraft over the course of the campaign).

As always, simplicity vs diversity

**Comment by Wollivan** [comment]
Yeah good point, I don't want to sacrifice simplicity at the expense of new players. But also don't want to make the game unexciting to players looking for more.

Quite the conundrum!

**Comment by asmialek** [comment]
I have two new traits:
- **High Service Ceiling**  - This plane can have up to 6 altitude tokens. _(Now that I read about some French planes, they can fly at quite high maximum altitude in comparison to German or British ones, but maybe I'm just comparing the models from different periods...)_
- **Carrier Fighter** - When deploying this plane can be placed within 4 straight maneuver templates from the edge. _(This is a nod to a bunch of post-WW1 Japanese planes, which were used from aircraft carriers.)_
- **_Something something reinforced hull?_** - This plane can reroll 1 additional die when defending.

If anyone has any ideas for traits feel free to share!

**Comment by Wollivan** [comment]
This could all get out of hand (in a good way) with the addition of non plane pieces such as aircraft carriers, immovable but controllable anti air guns... The possibilities are endless!
